crates/aiken-project/src/blueprint/validator.rs

```diff
@@ -1096,4 +1156,266 @@ mod test {
             }),
         )
     }
+
```

Comment:
Newly introduced validations tests below. The `fixture_definitions` is a minimal blueprint schema with definitions for `Int`, `ByteArray` and `Bool`, reuse across tests; other necessary definitions are defined inline in each tests.